python/python/ert/ecl/ecl_file.py

```diff
@@ -468,53 +389,7 @@ def iget_kw( self , index , copy = False):
 
 
     def iget_named_kw( self , kw_name , index , copy = False):
-        """
-        Will return EclKW nr @index reference with header @kw_name.
-
-        The keywords in a an ECLIPSE file are organized in a long
-        linear array; keywords with the same name can occur many
-        times. For instance a summary data[1] file might look like
-        this:
-
-           SEQHDR
-           MINISTEP
-           PARAMS
-           MINISTEP
-           PARAMS
-           MINISTEP
-           PARAMS
-           ....
-
-        To get the third 'PARAMS' keyword you can use the method call:
-
-            params_kw = file.iget_named_kw( "PARAMS" , 2 )
-
-        The functionality of the iget_named_kw() method is also
-        available through the __getitem__() method as:
-
-           params_kw = file["PARAMS"][2]
-
-        Observe that the returned EclKW instance is only a reference
-        to the data owned by the EclFile instance.
-
-        Observe that syntactically this is equivalent to
-        file[kw_name][index], however the latter form will imply that
-        all the keywords of this type are loaded from the file. If you
-        know that only a few will actually be used it will be faster
-        to use this method.
-
-        [1]: For working with summary data you are probably better off
-             using the EclSum class.
-        """
-        if index < self.num_named_kw( kw_name ):
-            kw = self.__iget_named( kw_name , index )
-            if copy:
-                return EclKW.copy( kw )
-            else:
-                return kw
-        else:
-            raise KeyError('Asked for occurence:%d of %d of keyword "%s".'
-                           % (index  , self.num_named_kw( kw_name ), kw_name))
+        return self.global_view.iget_named_kw( kw_name , index , copy )
```

Comment:
Sorry, I meant `self.global_view[kw_name][index]`.  However, I don't know what to make of `copy`.